datasets/trecqa.py

```diff
@@ -14,10 +15,14 @@ class TRECQA(CastorPairDataset):
     NAME = 'trecqa'
     NUM_CLASSES = 2
     ID_FIELD = Field(sequential=False, tensor_type=torch.FloatTensor, use_vocab=False, batch_first=True)
+    AID_FIELD = Field(sequential=False, use_vocab=False, batch_first=True)
     TEXT_FIELD = Field(batch_first=True, tokenize=lambda x: x)  # tokenizer is identity since we already tokenized it to compute external features
-    EXT_FEATS_FIELD = Field(tensor_type=torch.FloatTensor, use_vocab=False, batch_first=True, tokenize=lambda x: x)
+    EXT_FEATS_FIELD = Field(tensor_type=torch.FloatTensor, use_vocab=False, batch_first=True, tokenize=lambda x: x,
+                            postprocessing=Pipeline(lambda arr, _, train: [float(y) for y in arr]))
```

Comment:
Did you try setting `sequential` to `False`. I'm wondering maybe if you do that then you no longer need to `postprocessing` argument.

See this: https://github.com/pytorch/text/blob/v0.2.0/torchtext/data/field.py#L294. If `sequential` is false then torchtext seems to do the conversion for you.

Reply:
I will get an error if I delete the `postprocessing` argument and add `sequential=False`:
```
for batch_idx, batch in enumerate(iter(train_iter)): 
.....
TypeError: only length-1 arrays can be converted to Python scalars
```
I use torchtext v0.2 and pytorch 0.2.0 now.